src/VCF_Tasks.cpp

```diff
@@ -61,10 +65,39 @@ bool init_buzzer_control_task(const unsigned long& sysMicros, const HT_TASK::Tas
 }
 bool run_buzzer_control_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
 {
-    digitalWrite(BUZZER_CONTROL_PIN, vcr_system_data.buzzer_is_active);
+    digitalWrite(BUZZER_CONTROL_PIN, vcf_data.system_data.buzzer_is_active);
+
+    Serial.println("running");
     
     return true;
 }
 HT_TASK::Task buzzer_control_task = HT_TASK::Task(init_buzzer_control_task, run_buzzer_control_task, 10, 1000UL); // 1000us is 1kHz //NOLINT
 
+bool run_send_vcf_data_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
+{
+
+    vcf_data.system_data.pedals_system_data.accel_percent = (taskInfo.executions % 100) / 100.0f;
+
+    hytech_msgs_VCFData_s protoc_struct = VCFEthernetInterface::make_vcf_data_msg(vcf_data);
+
+    handle_ethernet_socket_send_pb<hytech_msgs_VCFData_s, hytech_msgs_VCFData_s_size>(debug_ip, VCF_SEND_PORT, &protobuf_send_socket, protoc_struct, &hytech_msgs_VCFData_s_msg);
+
+    Serial.printf("tried to send %f\n", vcf_data.system_data.pedals_system_data.accel_percent);
+
+    return true;
+}
+HT_TASK::Task send_vcf_data_task = HT_TASK::Task(HT_TASK::DUMMY_FUNCTION, run_send_vcf_data_task, 11, 50000UL); // 1 000 000 us is 1Hz //NOLINT
+
+bool run_recv_vcr_data_task(const unsigned long& sysMicros, const HT_TASK::TaskInfo& taskInfo)
+{
+    etl::optional<hytech_msgs_VCRData_s> protoc_struct = handle_ethernet_socket_receive<hytech_msgs_VCRData_s_size, hytech_msgs_VCRData_s>(&protobuf_recv_socket, &hytech_msgs_VCRData_s_msg);
+    if (protoc_struct)
+    {
+        Serial.println("Received protobuf message!");
+        
+    }
```

Comment:
this task should read the buzzer_is_active value from VCR and assign it to the buzzer_is_active field of vcf_data (so the buzzer control task can digitalWrite to the buzzer)